sdks/java/io/parquet/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOTest.java

```diff
@@ -337,7 +337,7 @@ public void testReadFilesUnknownSchemaFilesForGenericRecordThrowException() {
   public void testReadDisplayData() {
     DisplayData displayData = DisplayData.from(ParquetIO.read(SCHEMA).from("foo.parquet"));
 
-    Assert.assertThat(displayData, hasDisplayItem("filePattern", "foo.parquet"));
+    MatcherAssert.assertThat(displayData, hasDisplayItem("filePattern", "foo.parquet"));
```

Comment:
why the hamcrest one? the other one misses something? otherwise prefer the junit one.

Reply:
As per [JUnit documentation](https://junit.org/junit4/javadoc/latest/org/junit/Assert.html#assertThat(T,%20org.hamcrest.Matcher)) `Assert.assertThat` is now deprecated in favour of `MatcherAssert.assertThat`

Reply:
Oh I remember now, we have lots of those in the code base, ok then.